esp/src/eclwatch/ESPResult.js

```diff
@@ -77,6 +77,9 @@ define([
             }
         },
         preProcessResponse: function (response, request) {
+            if (response.Total == -1 || response.Total === 9223372036854776000 || response.Total === Number.MAX_VALUE) {
```

Comment:
Its a "common" JS number for this scenario and I reproduced it, it is essentially -1 converted to a +ve integer and then rounded.
(I reproduced it and googled it)